### DISCUSSION TITLE
Anyone have next working with yarn workspaces?

### DISCUSSION THREAD

**Discussion by special-character** [vercel/vercel#3890] [General] [answered]
I looked at the example here and tried it out myself: https://github.com/zeit/next.js/tree/canary/examples/with-yarn-workspaces

I got the error: `Module not found: Can't resolve 'bar'` which is the same error as the GitHub issue here: https://github.com/zeit/next.js/issues/10734

I pulled it locally and ran it without any problem but CI can't seem to find local packages from yarn workspaces. That makes sense on some level because the root of the next app is a single package and CI only looks at that package when doing `yarn` and building.

Thanks for any help!

**Comment by paulogdm** [comment]
Hello @special-character ,

Thank you very much for trying the new "Discussions" feature with us. Unfortunately, we do not support "monorepo" in a good way yet. Even if possible today with some configuration (builds + routes), I recommend that you wait for official support. We will update this discussion once this feature lands.

**Comment by special-character** [comment, marked as answer]
Hey @paulogdm, I submitted a PR to fix the `with-yarn-workspaces` in Next example that worked for me for now.

Thanks for all of the work and response!

**Reply by special-character** [reply]
PR if relevant: https://github.com/zeit/next.js/pull/10876

**Reply by paulogdm** [reply]
Thanks for the PR!
